models/user.dao.js

```diff
@@ -0,0 +1,52 @@
+const { AppDataSource } = require("./dataSource");
+
+  const createuser = async (email, password, firstname, lastname, skintype) => {
+    
+    try{
+   
+    return AppDataSource.query(
+      `
+      INSERT INTO users (
+      email,
+      password, 
+      first_name,
+      last_name,
+      skin_type
+    ) VALUES (
+      ?,
+      ?,
+      ?,
+      ?,
+      ?
+    )
+    `, 
+    [email, password, firstname, lastname, skintype]
+    );
+    }  catch(err) {
+    const error= new Error('INVAID_DATA_INPUT');
+    error.statusCode = 500;
+    throw error;
+  }
+  
+};
+
+
+const connectuser = async (email) => {
```

Comment:
함수나 변수명은 그 대상의 행위와 예측 결과를 알 수 있도록 작명해주는 것이 좋습니다.

현재 해당 함수는 user의 이메일 정보를 받아 유저의 데이터를 확인하는 함수이기 때문에 아래와 같이 수정해주세요.

```suggestion
const getUserrByEmail = async (email) => {
```